.ci/jenkins/Jenkinsfile.prod.nightly

```diff
@@ -258,8 +258,10 @@ pipeline {
                             def sanitizedProjectName = f.replaceAll('/', '_').replaceAll('-', '_')
                             def buildScript = env["PME_BUILD_SCRIPT_${sanitizedProjectName}"]
                             def pomPath = maven.getProjectPomFromBuildCmd(buildScript)
-                            if (new File(pomPath).exists()) {
```

Comment:
`new File()` approach does not work as it would try to find the file in Jenkins master and not in the running node.